src/components/UI/Card/ProductCards/ProductCards.jsx

```diff
@@ -66,7 +77,7 @@ function ProductCards(props) {
   return (
     <>
         <div className="card cart-card">
-            <img className="card-img" src={productImage} alt={author} />
+            <img className="card-img" src={productImage} alt={author} loading="lazy"/>
 
             <div className="card-content">
                 <div className="card-body">
```

Comment:
you have kept title in card-body className and body in card-title className, You should change it

Reply:
ok